### ISSUE TITLE
Game crashes on render distance greater than 32 chunks with Nvidium

### ISSUE THREAD

**Issue by LemonHeadOnGit** [qouteall/ImmersivePortalsMod#1442]
### Is it a mod compatibility issue?

No

### Minecraft version

1.20.1

### Describe the issue

When paired with a render distance greater than 32 chunks (easily achieved when using Nvidium), the game crashes with an error message stating: `java.lang.IllegalArgumentException: chunk loader radius cannot be larger than 32`.

**Comment by qouteall** [comment]
https://github.com/iPortalTeam/ImmersivePortalsMod/issues/1359

**Comment by qouteall** [comment]
Does any rendering issue occur when render distance is not larger than 32?

**Comment by LemonHeadOnGit** [comment]
Yes, particles and certain entities are in fact visible through blocks. With shaders, it sometimes leads to entities glowing when they're not supposed to.

Apolgies for not seeing the pre-existing issue!

**Comment by LemonHeadOnGit** [comment]
The effects vary depending on the postion of the camera

**Comment by Mythfat** [comment]
Some blocks get extremely zoomed in and might appear to come out of the portal itself. Depends on how far or close you are to the portal.
1.20.1
16(vanilla) + 64(nvidium) render distance (16 simulation distance)
